File: tasty-fast/src/main/java/com/misha/tastyfast/requests/storeRequests/StoreResponse.java
package com.misha.tastyfast.requests.storeRequests;

import com.misha.tastyfast.feedback.req.FeedBackResponse;
import com.misha.tastyfast.requests.drinkRequests.DrinksResponse;
import com.misha.tastyfast.requests.orderRequests.OrderResponse;
import com.misha.tastyfast.requests.productRequests.ProductResponse;
import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class StoreResponse {
    private Integer id;
    private String storeName;
    private String address;
    private String phoneNumber;
    private String email;
    private String description;
    private String openingHours;
    private Double rating;
    private boolean isActive;
    private Boolean deliveryAvailable;
    private String websiteUrl;
    private Integer ownerId;
    private byte[] logoUrl;

    private List<ProductResponse> productResponses;
    private List<DrinksResponse> drinksResponses;
    private List<FeedBackResponse> feedBackResponses;
    private List<OrderResponse> orderResponses;

}

File: tasty-fast-ui/src/app/services/models/store-response.ts
/* tslint:disable */
/* eslint-disable */
import { DrinksResponse } from '../models/drinks-response';
import { FeedBackResponse } from '../models/feed-back-response';
import { OrderResponse } from '../models/order-response';
import { ProductResponse } from '../models/product-response';
export interface StoreResponse {
  active?: boolean;
  address?: string;
  deliveryAvailable?: boolean;
  description?: string;
  drinksResponses?: Array<DrinksResponse>;
  email?: string;
  feedBackResponses?: Array<FeedBackResponse>;
  id?: number;
  logoUrl?: string;
  openingHours?: string;
  orderResponses?: Array<OrderResponse>;
  ownerId?: number;
  phoneNumber?: string;
  productResponses?: Array<ProductResponse>;
  rating?: number;
  storeName?: string;
  websiteUrl?: string;
}
